### PR TITLE
Allow for ifos option with default hdf dataset names

### PR DESCRIPTION
**Pull request by tdent**
As we probably won't need to specify the 'columns' in future, use default correct dataset names derived from ifos.

### CODE REVIEW

**Comment by tdent** [comment]
Ping on this ..

File: bin/pycbc_optimal_snr [hunk 1]
```diff
@@ -171,7 +175,13 @@ if __name__ == '__main__':
                              'injections. Format PART/NUM_PARTS')
     opts = parser.parse_args()
 
-    detectors = opts.snr_columns.keys()
+    if opts.ifos is not None:
+        detectors = opts.ifos
+        if opts.snr_columns is not None:
+            raise RuntimeError("Can't use both --ifos and --snr-columns !")
```
Comment (titodalcanton) [inline bin/pycbc_optimal_snr:181]:
```suggestion
            parser.error("Can't use both --ifos and --snr-columns !")
```

**Review by titodalcanton — APPROVED** [review]
Seems ok to me, I just made a minor suggestion for a better user interface.

**Comment by titodalcanton** [comment]
@tdent it looks like this may need a rebase on baster to pass the tests.

**Comment by tdent** [comment]
Sigh. I also lost track of why this wasn't merged earlier.

**Pull request merged**